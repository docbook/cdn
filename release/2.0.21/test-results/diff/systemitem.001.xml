<html xmlns:deltaxml="http://www.deltaxml.com/ns/well-formed-delta-v1" xmlns="http://www.w3.org/1999/xhtml" xmlns:dxx="http://www.deltaxml.com/ns/xml-namespaced-attribute" xmlns:dxa="http://www.deltaxml.com/ns/non-namespaced-attribute" deltaxml:deltaV2="A!=B" deltaxml:version="2.0" deltaxml:content-type="full-context">
   <head deltaxml:deltaV2="A=B">
      <link href="../../resources/css/db-prism.css" rel="stylesheet" type="text/css"/>
      <link rel="stylesheet" type="text/css" href="../../resources/css/default.css"/>
      <link href="../../resources/css/prism.css" rel="stylesheet" type="text/css"/>
      <meta charset="utf-8"/>
      <meta name="generator" content="DocBook XSL 2.0 Stylesheets Vxxx"/>
      <script type="text/javascript" src="../../resources/js/dbmodnizr.js"/>
      <title>Article wrapper</title>
   </head>
   <body deltaxml:deltaV2="A!=B">
      <article deltaxml:deltaV2="A!=B" id="R.1" class="article">
         <header deltaxml:deltaV2="A=B" class="article-titlepage">
            <h2>Article wrapper</h2>
         </header>
         <p deltaxml:deltaV2="A">There are several classes of “<code class="tag-element">systemitem</code>”:</p><div deltaxml:deltaV2="B" class="content">
            <p>There are several classes of “<code class="tag-element">systemitem</code>”:</p>
            <table border="0" summary="Simple list" class="simplelist">
               <tr>
                  <td>
                     <span class="member">
                        <code class="systemitem">systemitem</code>
                     </span>
                  </td>
               </tr>
               <tr>
                  <td>
                     <span class="member">
                        <code id="daemon" class="systemitem daemon">daemon systemitem</code>
                     </span>
                  </td>
               </tr>
               <tr>
                  <td>
                     <span class="member">
                        <code id="domainname" class="systemitem domainname">domainname systemitem</code>
                     </span>
                  </td>
               </tr>
               <tr>
                  <td>
                     <span class="member">
                        <code id="etheraddress" class="systemitem etheraddress">etheraddress systemitem</code>
                     </span>
                  </td>
               </tr>
               <tr>
                  <td>
                     <span class="member">
                        <code id="event" class="systemitem event">event systemitem</code>
                     </span>
                  </td>
               </tr>
               <tr>
                  <td>
                     <span class="member">
                        <code id="eventhandler" class="systemitem eventhandler">eventhandler systemitem</code>
                     </span>
                  </td>
               </tr>
               <tr>
                  <td>
                     <span class="member">
                        <code id="filesystem" class="systemitem filesystem">filesystem systemitem</code>
                     </span>
                  </td>
               </tr>
               <tr>
                  <td>
                     <span class="member">
                        <code id="fqdomainname" class="systemitem fqdomainname">fqdomainname systemitem</code>
                     </span>
                  </td>
               </tr>
               <tr>
                  <td>
                     <span class="member">
                        <code id="groupname" class="systemitem groupname">groupname systemitem</code>
                     </span>
                  </td>
               </tr>
               <tr>
                  <td>
                     <span class="member">
                        <code id="ipaddress" class="systemitem ipaddress">ipaddress systemitem</code>
                     </span>
                  </td>
               </tr>
               <tr>
                  <td>
                     <span class="member">
                        <code id="library" class="systemitem library">library systemitem</code>
                     </span>
                  </td>
               </tr>
               <tr>
                  <td>
                     <span class="member">
                        <code id="macro" class="systemitem macro">macro systemitem</code>
                     </span>
                  </td>
               </tr>
               <tr>
                  <td>
                     <span class="member">
                        <code id="netmask" class="systemitem netmask">netmask systemitem</code>
                     </span>
                  </td>
               </tr>
               <tr>
                  <td>
                     <span class="member">
                        <code id="newsgroup" class="systemitem newsgroup">newsgroup systemitem</code>
                     </span>
                  </td>
               </tr>
               <tr>
                  <td>
                     <span class="member">
                        <code id="osname" class="systemitem osname">osname systemitem</code>
                     </span>
                  </td>
               </tr>
               <tr>
                  <td>
                     <span class="member">
                        <code id="process" class="systemitem process">process systemitem</code>
                     </span>
                  </td>
               </tr>
               <tr>
                  <td>
                     <span class="member">
                        <code id="protocol" class="systemitem protocol">protocol systemitem</code>
                     </span>
                  </td>
               </tr>
               <tr>
                  <td>
                     <span class="member">
                        <code id="resource" class="systemitem resource">resource systemitem</code>
                     </span>
                  </td>
               </tr>
               <tr>
                  <td>
                     <span class="member">
                        <code id="server" class="systemitem server">server systemitem</code>
                     </span>
                  </td>
               </tr>
               <tr>
                  <td>
                     <span class="member">
                        <code id="service" class="systemitem service">service systemitem</code>
                     </span>
                  </td>
               </tr>
               <tr>
                  <td>
                     <span class="member">
                        <code id="systemname" class="systemitem systemname">systemname systemitem</code>
                     </span>
                  </td>
               </tr>
               <tr>
                  <td>
                     <span class="member">
                        <code id="username" class="systemitem username">username systemitem</code>
                     </span>
                  </td>
               </tr>
            </table>
         </div><deltaxml:textGroup deltaxml:deltaV2="A"><deltaxml:text deltaxml:deltaV2="A">
         </deltaxml:text></deltaxml:textGroup><table deltaxml:deltaV2="A" border="0" summary="Simple list" class="simplelist">
            <tr>
               <td>
                  <span class="member">
                     <code class="systemitem">systemitem</code>
                  </span>
               </td>
            </tr>
            <tr>
               <td>
                  <span class="member">
                     <code id="daemon" class="systemitem daemon">daemon systemitem</code>
                  </span>
               </td>
            </tr>
            <tr>
               <td>
                  <span class="member">
                     <code id="domainname" class="systemitem domainname">domainname systemitem</code>
                  </span>
               </td>
            </tr>
            <tr>
               <td>
                  <span class="member">
                     <code id="etheraddress" class="systemitem etheraddress">etheraddress systemitem</code>
                  </span>
               </td>
            </tr>
            <tr>
               <td>
                  <span class="member">
                     <code id="event" class="systemitem event">event systemitem</code>
                  </span>
               </td>
            </tr>
            <tr>
               <td>
                  <span class="member">
                     <code id="eventhandler" class="systemitem eventhandler">eventhandler systemitem</code>
                  </span>
               </td>
            </tr>
            <tr>
               <td>
                  <span class="member">
                     <code id="filesystem" class="systemitem filesystem">filesystem systemitem</code>
                  </span>
               </td>
            </tr>
            <tr>
               <td>
                  <span class="member">
                     <code id="fqdomainname" class="systemitem fqdomainname">fqdomainname systemitem</code>
                  </span>
               </td>
            </tr>
            <tr>
               <td>
                  <span class="member">
                     <code id="groupname" class="systemitem groupname">groupname systemitem</code>
                  </span>
               </td>
            </tr>
            <tr>
               <td>
                  <span class="member">
                     <code id="ipaddress" class="systemitem ipaddress">ipaddress systemitem</code>
                  </span>
               </td>
            </tr>
            <tr>
               <td>
                  <span class="member">
                     <code id="library" class="systemitem library">library systemitem</code>
                  </span>
               </td>
            </tr>
            <tr>
               <td>
                  <span class="member">
                     <code id="macro" class="systemitem macro">macro systemitem</code>
                  </span>
               </td>
            </tr>
            <tr>
               <td>
                  <span class="member">
                     <code id="netmask" class="systemitem netmask">netmask systemitem</code>
                  </span>
               </td>
            </tr>
            <tr>
               <td>
                  <span class="member">
                     <code id="newsgroup" class="systemitem newsgroup">newsgroup systemitem</code>
                  </span>
               </td>
            </tr>
            <tr>
               <td>
                  <span class="member">
                     <code id="osname" class="systemitem osname">osname systemitem</code>
                  </span>
               </td>
            </tr>
            <tr>
               <td>
                  <span class="member">
                     <code id="process" class="systemitem process">process systemitem</code>
                  </span>
               </td>
            </tr>
            <tr>
               <td>
                  <span class="member">
                     <code id="protocol" class="systemitem protocol">protocol systemitem</code>
                  </span>
               </td>
            </tr>
            <tr>
               <td>
                  <span class="member">
                     <code id="resource" class="systemitem resource">resource systemitem</code>
                  </span>
               </td>
            </tr>
            <tr>
               <td>
                  <span class="member">
                     <code id="server" class="systemitem server">server systemitem</code>
                  </span>
               </td>
            </tr>
            <tr>
               <td>
                  <span class="member">
                     <code id="service" class="systemitem service">service systemitem</code>
                  </span>
               </td>
            </tr>
            <tr>
               <td>
                  <span class="member">
                     <code id="systemname" class="systemitem systemname">systemname systemitem</code>
                  </span>
               </td>
            </tr>
            <tr>
               <td>
                  <span class="member">
                     <code id="username" class="systemitem username">username systemitem</code>
                  </span>
               </td>
            </tr>
         </table>
      </article>
      <script deltaxml:deltaV2="A=B" src="../../resources/js/prism.js"/>
   </body>
</html>