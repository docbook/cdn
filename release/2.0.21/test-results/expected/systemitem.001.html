<html xmlns="http://www.w3.org/1999/xhtml"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8" /><title>Article wrapper</title><meta charset="utf-8" /><meta name="generator" content="DocBook XSL 2.0 Stylesheets V2.0.17" /><link rel="stylesheet" type="text/css" href="../../resources/css/default.css" /><script type="text/javascript" src="../../resources/js/dbmodnizr.js"></script><link href="../../resources/css/prism.css" rel="stylesheet" type="text/css" /><link href="../../resources/css/db-prism.css" rel="stylesheet" type="text/css" /></head><body><article id="R.1" class="article"><header class="article-titlepage"><h2>Article wrapper</h2></header><p>There are several classes of “<code class="tag-element">systemitem</code>”:</p><table border="0" summary="Simple list" class="simplelist"><tr><td><span class="member"><code class="systemitem">systemitem</code></span></td></tr><tr><td><span class="member"><code id="daemon" class="systemitem daemon">daemon systemitem</code></span></td></tr><tr><td><span class="member"><code id="domainname" class="systemitem domainname">domainname systemitem</code></span></td></tr><tr><td><span class="member"><code id="etheraddress" class="systemitem etheraddress">etheraddress systemitem</code></span></td></tr><tr><td><span class="member"><code id="event" class="systemitem event">event systemitem</code></span></td></tr><tr><td><span class="member"><code id="eventhandler" class="systemitem eventhandler">eventhandler systemitem</code></span></td></tr><tr><td><span class="member"><code id="filesystem" class="systemitem filesystem">filesystem systemitem</code></span></td></tr><tr><td><span class="member"><code id="fqdomainname" class="systemitem fqdomainname">fqdomainname systemitem</code></span></td></tr><tr><td><span class="member"><code id="groupname" class="systemitem groupname">groupname systemitem</code></span></td></tr><tr><td><span class="member"><code id="ipaddress" class="systemitem ipaddress">ipaddress systemitem</code></span></td></tr><tr><td><span class="member"><code id="library" class="systemitem library">library systemitem</code></span></td></tr><tr><td><span class="member"><code id="macro" class="systemitem macro">macro systemitem</code></span></td></tr><tr><td><span class="member"><code id="netmask" class="systemitem netmask">netmask systemitem</code></span></td></tr><tr><td><span class="member"><code id="newsgroup" class="systemitem newsgroup">newsgroup systemitem</code></span></td></tr><tr><td><span class="member"><code id="osname" class="systemitem osname">osname systemitem</code></span></td></tr><tr><td><span class="member"><code id="process" class="systemitem process">process systemitem</code></span></td></tr><tr><td><span class="member"><code id="protocol" class="systemitem protocol">protocol systemitem</code></span></td></tr><tr><td><span class="member"><code id="resource" class="systemitem resource">resource systemitem</code></span></td></tr><tr><td><span class="member"><code id="server" class="systemitem server">server systemitem</code></span></td></tr><tr><td><span class="member"><code id="service" class="systemitem service">service systemitem</code></span></td></tr><tr><td><span class="member"><code id="systemname" class="systemitem systemname">systemname systemitem</code></span></td></tr><tr><td><span class="member"><code id="username" class="systemitem username">username systemitem</code></span></td></tr></table></article><script src="../../resources/js/prism.js"></script></body></html>